backend/spiritual_avatar_generation_engine.py

```
@@ -16,6 +16,8 @@
 from fastapi import HTTPException, Depends
 import asyncpg
 
+# REFRESH.MD: Use relative imports for deployment robustness
+from ..services.supabase_storage_service import SupabaseStorageService, get_storage_service
```

Ruff (0.12.2)

`..services.supabase_storage_service.SupabaseStorageService` imported but unused

Remove unused import

(F401)